style.css

```diff
@@ -0,0 +1,431 @@
+ body {
+     background-color: #f9f9f9;
+     font-family: Roboto, Arial, sans-serif;
+     box-sizing: border-box;
+ }
+ 
+ .flex-container {
+     display: flex;
+     flex-direction: row;
+     justify-content: space-between;
+     align-items: center;
+     padding-top: 10px;
+     height: 40px;
+     background-color: white;
+     position: relative;
+ }
+ 
+ .fa-bars {
+     position: absolute;
+     left: 10px;
+ }
+ 
+ .logo {
+     width: 60px;
+     height: 15px;
+     margin-left: 40px;
+ }
+ 
+ .myInput {
+     width: 430px;
+ }
+ 
+ .fa-search {
+     border: 1px solid #808080;
+     background-color: #f8f8f8;
+     padding: 2px 12px 2px 12px;
+     position: relative;
+     top: 1.5px;
+ }
+ /*
+.search-icon {
+    position: absolute;
+    left: 630px;
+}
+
+*/
+ 
+ .fa-upload {
+     color: #808080;
+ }
+ 
+ .sign {
+     /*
+    position: absolute;
+    right: 10px;
+    */
+     background-color: #167AC6;
+     color: white;
+     border: solid 1px #167AC6;
+     border-radius: 2px;
+     margin-right: 10px;
+ }
+ 
+ .mini-banner {
+     display: flex;
+     flex-direction: row;
+     justify-content: space-around;
+     background-color: #167AC6;
+     color: white;
+     margin-top: 10px;
+     height: 30px;
+     position: relative;
+ }
+ 
+ .fa-user {
+     position: absolute;
+     bottom: 7px;
+     left: 10px;
+ }
+ 
+ h6 {
+     position: absolute;
+     left: 40px;
+     top: -15px;
+ }
+ 
+ .learn_more {
+     position: absolute;
+     right: 30px;
+     bottom: 5px;
+     color: white;
+     text-decoration: none;
+ }
```

Comment:
- [ ] You are repeating the same rules in different parts of your CSS, such as (`display: flex;`, `position: absolute;`, `background-color: white;`, among others. For good coding practices ([check this link](https://speckyboy.com/good-bad-css-practices)), follow the DRY rule (Don't repeat yourself).
![image](https://user-images.githubusercontent.com/53256139/89356769-33203f80-d684-11ea-8a2b-7776e5a73811.png)

  👉 Here you have an explanation of how you can implement it:

  You can use multiple classes for any HTML element, so for example,

  If you use these lines in many classes:
  ```
  color: #fff;
  background-color: #000;
  ```
  and you use it for .class1, .class2, and .class3 
  you can put them inside a single class called for example "common-class" so it will look like this:
  **CSS**
  ```
  .class1 {
    font-size: 24px;
  }
  .class2 {
    font-size: 30px;
  }
  .class3 {
    font-size: 10px;
  }
  .common-class {
    color: #fff;
    background-color: #000
  }
  ```
  and in **HTML**:
  ```
  <nav class="class1 common-class">  <!-- this will have font size of 24px, color of #fff and background-color of #000 -->
  ...
  <div class="class2 common-class">  <!-- this will have font size of 30px, color of #fff and background-color of #000 -->
  ...
  <footer class="class3 common-class">   <!-- this will have font size of 10px, color of #fff and background-color of #000 -->
  ...
  ```